SteamKit2/SteamKit2/Steam/SteamClient/Configuration/SteamConfigurationBuilder.cs

```diff
@@ -35,7 +35,7 @@ public static SteamConfigurationState CreateDefaultState()
 
                 MachineInfoProvider = MachineInfoProvider.GetDefaultProvider(),
 
-                ProtocolTypes = ProtocolTypes.Tcp,
+                ProtocolTypes = ProtocolTypes.WebSocket,
```

Comment:
```suggestion
                ProtocolTypes = ProtocolTypes.Tcp | ProtocolTypes.WebSocket,
``` since not all the tcp servers are bad